src/Wallabag/ApiBundle/Controller/EntryRestController.php

```diff
@@ -570,18 +570,31 @@ public function patchEntriesReloadAction(Entry $entry)
      * @ApiDoc(
      *      requirements={
      *          {"name"="entry", "dataType"="integer", "requirement"="\w+", "description"="The entry ID"}
+     *      },
+     *      parameters={
+     *          {"name"="expect", "dataType"="string", "required"=false, "format"="id or entry", "description"="Only returns the id instead of the deleted entry's full entity if 'id' is specified. Default to entry"},
      *      }
      * )
      *
      * @return JsonResponse
      */
-    public function deleteEntriesAction(Entry $entry)
+    public function deleteEntriesAction(Entry $entry, Request $request)
     {
+        $expect = $request->query->get('expect') ?: 'entry';
```

Comment:
This should be enough:
```suggestion
        $expect = $request->query->get('expect', 'entry');
```

Reply:
Thanks :+1: 
(I could have used the commit suggestion feature of github but I saw it after my push)